### PR TITLE
Automatic Python 3.5 transpilation

### PR DESCRIPTION
**Pull request by rahularya50**
To use:
 - Run `bash strip_annotations`
 - Take the `compiled` folder, put it in the target directory, and rename it `editor`

### CODE REVIEW

File: editor/ok_interface.py [hunk 1]
```diff
@@ -17,13 +17,6 @@
 
 FAILURE_SETUP_FOOTER = "; Raw ok output over"
 
-##############
```
Comment (kavigupta) [inline editor/ok_interface.py:20]: Why these changes?

File: editor/log.py [hunk 2]
```diff
@@ -103,8 +103,8 @@ def setroot(cls, root: Holder):
 def silence(*args): pass
 
 
-def print_announce(message, local, root):
```
Comment (kavigupta) [inline editor/log.py:106]: Should we just delete this if its not being used?

**Review by kavigupta — CHANGES_REQUESTED** [review]

File: editor/ok_interface.py (hunk 1 above)
Comment (rahularya50) [reply editor/ok_interface.py:20]: `py-backwards` messes up the order of imports for some reason, and moved them before the path is updated, so the imports would fail. I moved the imports inside the function call, and now it works correctly.

File: editor/log.py (hunk 2 above)
Comment (rahularya50) [reply editor/log.py:106]: Guess so. It was useful for debugging at one point, but I think the web interface is sufficiently stable now so this is no longer necessary.

File: editor/ok_interface.py (hunk 1 above)
Comment (kavigupta) [reply editor/ok_interface.py:20]: Huh interesting.

**Review by kavigupta — APPROVED** [review]
